package.json

```diff
@@ -179,13 +182,16 @@
     "csso": "^4.0.3",
     "electron": "^8.2.5",
     "electron-builder": "^22.6.0",
+    "electron-devtools-installer": "^3.0.0",
     "electron-notarize": "^0.3.0",
-    "eslint": "^6.8.0",
+    "eslint": "^7.0.0",
     "eslint-config-standard": "^14.1.1",
+    "eslint-loader": "^4.0.2",
     "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.0.1",
+    "eslint-plugin-vue": "^7.0.0-alpha.1",
```

Comment:
I'm not really comfortable with adding alphas (or betas, for that matter) as dependencies, and tend to wait until the plugins become stable — is there a reason we need the alpha here?

Reply:
This is necessary for eslint 7 support. Should I downgrade both? (One could argue that the alpha is not a real problem as it is only a dev dependency anyway and there are no problems right now...but I share your sentiment to stick to stable releases.)

Reply:
Mh … do you think they'll upgrade to stable quickly? Because ESLint 7 sounds good, so I'll trust your expectations regarding this.

Reply:
The plugin seems to be developed pretty actively. I would keep the alpha for now (with the option to downgrade in case of problems).

Reply:
Perfect! :)